configs/rt_slam_plugins.yaml

```diff
@@ -5,5 +5,9 @@ plugin_group:
     path:
       git_repo: https://github.com/ILLIXR/Kimera-VIO
       version: 6ea99d33fc2435a8c514049416dcb2f8a0e31692
+  #- name: OpenVINS
+  #  path:
+  #    git_repo: https://github.com/ILLIXR/open_vins.git
+  #    version: c6a4f8835851d21f87add38260e73df63c6354a1
```

Comment:
@Hyjale can you remove this. We decided to not have openvins in the config file. A user can add this themselves.

@mhuzai @charmoniumQ unless we want to change this.

Reply:
We had a separate file for OV before.

Reply:
We can just let this through now and change it later.